src/gridstack-dd.ts

```diff
@@ -449,6 +449,13 @@ GridStack.prototype._prepareDragDropByNode = function(node: GridStackNode): Grid
     this.engine.moveNode(node, x, y, w, h);
     if (resizing && node.subGrid) { (node.subGrid as GridStack).onParentResize(); }
     this._updateContainerHeight();
+
+    if (this._gsEventHandler[event.type]) {
+      let target: GridItemHTMLElement = event.target as GridItemHTMLElement;

```

Comment:
is target != node.el ?
I need to think about are we calling about placeholder (which is grid constrained) or the actual content is that is freely being resized by the mouse (pixel following)... I would have to debug it and see what makes sense. if you are doing the live item when it won't match the mouse...

Also I would not conditionally call _writePosAttr() if you have an event as it's changes the behavior and hard to debug/test. Always do this (assuming we need that - which I don't believe we do on the placeholder as it need to reflect the new column/row sizing.

Reply:
Target == node.el, but from node.el you can't go back to el.gridstackNode. Probably because it's a recursive reference.

Reply:
node.el.gridstackNode should work - I should always have that circular link, but placeholder vs real widget being dragged/resized might be different. Again I'll have to look.